modules/openapi-generator/src/test/java/org/openapitools/codegen/options/ElixirClientOptionsProvider.java

```diff
@@ -47,6 +48,7 @@ public Map<String, String> createOptions() {
                 .put(CodegenConstants.LEGACY_DISCRIMINATOR_BEHAVIOR, "true")
                 .put(CodegenConstants.DISALLOW_ADDITIONAL_PROPERTIES_IF_NOT_PRESENT, "true")
                 .put(CodegenConstants.ENUM_UNKNOWN_DEFAULT_CASE, ENUM_UNKNOWN_DEFAULT_CASE_VALUE)
+                .put(CodegenConstants.REMOVE_ENUM_VALUE_PREFIX, REMOVE_ENUM_VALUE_PREFIX_VALUE)
```

Comment:
Did none of the samples change for any of these generators when you added the option?

Do they properly support this option?

Reply:
This new option was implemented in DefaultCodegen and has been in use for a long time.
My intention here is not to develop a new feature but to make its configuration available via configOptions so that devs can find this option more easily.
I just received test errors and resolved them.